### PR TITLE
Make tests pass on CircleCI

### PR DESCRIPTION
**Pull request by monikasulik**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -55,10 +55,10 @@ SET_LOCAL_LOCUST_SSO := \
 
 # TODO: set these to docker network names when docker works fully
 SET_LOCAL_PYTEST_ENV_VARS := \
-	export DIRECTORY_API_URL=http://directory_api_webserver:8000/; \
```
Comment (konradko) [inline makefile]: These env vars are for running against the locally provisioned environment. You can give a different command in `circle.yml` if needed.

File: tests/locust/test_api.py [hunk 2]
```diff
@@ -1,130 +0,0 @@
...
-
-
-class PublicPagesAPI(TaskSet):
-
-    @task
-    def health(self):
-        self.client.get(get_relative_url('api:health'))
-
-
-class AuthenticatedPagesAPI(TaskSet):
```
Comment (konradko) [inline tests/locust/test_api.py]: Do we have coverage of all these endpoints in `ui-buyer` tests now?

File: makefile (hunk 1 above)
Comment (monikasulik) [inline makefile]: I get the same errors both on local and on circleCI, hence figured it made sense to change this on both. Does it work for you locally?

File: tests/locust/test_api.py (hunk 2 above)
Comment (monikasulik) [inline tests/locust/test_api.py]: Nope, working on it.

**Pull request closed without merging**